addons/website_slides/views/slide_slide_views.xml

```diff
@@ -68,11 +68,14 @@
                             </page>
                             <page string="Document">
                                 <group>
-                                    <field name="url" attrs="{'required': [('image', '=', False)]}"/>
+                                    <field name="url"
+                                        attrs="{'required': [('image', '=', False)], 'invisible': [('slide_type', 'not in', ('document', 'presentation', 'video'))]}" />
```

Comment:
can you please coordonnate yourself with another guy that chipotated with those at https://github.com/odoo/odoo/pull/31097/commits/986267c67d3c0a60c38315775721074a75fd621b
Histoire d'uniformiser et qu'on ait le meme commit. Premier qui merge a gagné, as usual.

Reply:
I coordinated my chipoterie with the concerned dude sir.